Makefile

```diff
@@ -112,6 +112,18 @@ pause-container-release: pause-container
 	mkdir -p "$(shell pwd)/out"
 	@docker save ${PAUSE_CONTAINER_IMAGE}:${PAUSE_CONTAINER_TAG} > "$(shell pwd)/out/${PAUSE_CONTAINER_TARBALL}"
 
+# Variable to determine branch/tag of amazon-ecs-cni-plugins
+revision=master
+cni-sources:
+	@git clone https://github.com/aws/amazon-ecs-cni-plugins.git --branch $(revision)
+
+cni-plugins: cni-sources
```

Comment:
Rather than make `cni-plugins` depend on `cni-sources` (which means that I can't point it to a local build of of the plugins repo), can we have the `release` target depend on that instead?